scripts/openmc-make-test-data

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python3
+
+"""
+Download ENDF/B-VII.1 ENDF and ACE files from NNDC and WMP files from GitHub and
+generate a full HDF5 library with incident neutron, incident photon, thermal
+scattering data, and windowed multipole data. This data is used for OpenMC's
+regression test suite.
+"""
+
+import glob
+import os
+from pathlib import Path
+import tarfile
+import tempfile
+from urllib.parse import urljoin
+import zipfile
+
+import openmc.data
+from openmc._utils import download
+
+base_ace = 'http://www.nndc.bnl.gov/endf/b7.1/aceFiles/'
```

Comment:
The urls in `base_ace` and `base_endf` appear to support https.  In general, it's preferable to use `https` protocol when possible.